includes/admin/import/class-batch-import.php

```diff
@@ -117,6 +112,34 @@ public function can_import() {
 		return (bool) apply_filters( 'edd_import_capability', current_user_can( $this->capability_type ) );
 	}
 
+	/**
+	 * Set the CSV file path. This then also parses the CSV and populates that property.
+	 *
+	 * @since 3.0
+	 * @param string $file
+	 *
+	 * @return array
+	 */
+	public function set_csv_file( $file ) {
```

Comment:
Nit-picky, but can we rename this method? I don't really like the word `set` because it's not setting `$this->csv` property, but returning the value. I think `get` or `make` would be better.